cumulus/polkadot-parachain/src/command.rs

```diff
@@ -54,7 +68,6 @@ enum Runtime {
 	CollectivesPolkadot,
 	CollectivesWestend,
```

Comment:
Could we further break this enum down?
I think `CollectivesPolkadot` and `CollectivesWestend` have the same codepaths.
Same for AssetHub, afaik only AssetHubPolkadot is different. Sow we could have `AssetHub` and `AssetHubPolkadot`.

Reply:
Yes, thanks ! Missed that. Done.